src/Illuminate/Mail/Mailables/Headers.php

```diff
@@ -95,7 +95,7 @@ public function text(array $text)
     public function referencesString(): string
     {
         return (new Collection($this->references))
-            ->map(fn ($messageId) => Str::wrap($messageId, '<', '>'))
+            ->map(fn ($messageId) => Str::of($messageId)->start('<')->finish('>'))
```

Comment:
This is not reversing, this is adding a `Stringable` instance overhead on each `$references` element.

Previous code used nested calls to `Str::start()` and `Str::finish()`

Reply:
Well, it reverses the functionality which is now asserted by the added test, not necessarily the syntax, which is more readable imo.